hasktorch/src/Torch/Functional.hs

```diff
@@ -455,10 +766,17 @@ conv2d' weight bias stride padding input =
         (1 :: Int) -- groups
         input
 
-solve :: Tensor -> Tensor -> (Tensor,Tensor)
+solve 
+    :: Tensor 
+    -> Tensor 
+    -> (Tensor,Tensor)
 solve b a = unsafePerformIO $ (cast2 ATen.solve_tt) b a
 
-cholesky_inverse :: Tensor -> Tri -> Tensor
+-- | Solves a linear system of equations with a positive semidefinite matrix to be inverted given its Cholesky factor matrix uu .
+cholesky_inverse
```

Comment:
@austinvhuang Should this also be camelCase? Similar to the choleskySolve change made in this PR?

Reply:
Yes it should be, just haven't gotten to all the bits to cleanup in this PR.

Basically the current informal convention is basically - code generated (such as the Functional/Internal module and everything in libtorch-ffi) is snake case, user facing is camelCase in line with Haskell convention, but there are still bits here and there to clean up (BCE is another one).

Feel free to add PR to fix other user-facing instances.

Reply:
Gotcha. Thanks. :)